packages/relayer/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.19.3
+
+RUN apt install git curl
+
+ADD https://github.com/ufoscout/docker-compose-wait/releases/download/2.9.0/wait /wait
+RUN chmod +x /wait
+
+RUN git clone --depth 1 https://github.com/taikoxyz/taiko-mono /taiko-mono
+
+WORKDIR /taiko-mono/packages/relayer
+
+RUN go mod download
+
+RUN CGO_ENABLED=0 GOOS=linux go build -o ./bin/relayer cmd/main.go
+
+ENTRYPOINT [ "./bin/relayer" ]
```

Comment:
Since this is a golang binary image, we can use two-stage-build?

Reply:
Sure, ill push that up